package.json

```diff
@@ -1,23 +1,22 @@
 {
-  "name": "monowingo",
+  "name": "@winglang/monorepo",
+  "private": true,
   "devDependencies": {
-    "@nrwl/devkit": "16.5.0",
     "bump-pack": "workspace:^",
     "changelogen": "^0.5.3",
-    "nx": "16.5.0",
-    "semver": "^7.3.8"
+    "semver": "^7.3.8",
+    "turbo": "^1.10.7"
   },
   "scripts": {
-    "build": "nx run-many --all --targets build,test,package",
-    "test": "nx run-many --all --target=test",
-    "package": "nx run-many --all --target=package",
-    "build:ci": "nx run-many --targets lint,build --all --configuration=release --output-style=stream --verbose",
-    "test:ci": "nx run-many --exclude=hangar --target=test --all --configuration=release --output-style=stream --verbose",
-    "package:ci": "nx run-many --target=package --all --configuration=release --output-style=stream --verbose",
+    "compile": "turbo compile --filter=!hangar",
+    "build": "turbo compile post-compile lint eslint test package --filter=!hangar",
+    "test": "turbo lint eslint test --filter=!hangar",
+    "package": "turbo package --filter=!hangar",
```

Comment:
I believe @eladb wanted to have root commands available that handled running everything (including hangar).

Reply:
What exact commands should we include? Currently:

- `build` executes everything
- `compile`, `test` and `package` execute the related subsets

Reply:
I think something like:

* `build` - full build: exactly what happens in CI


* `compile` - only compile
* `test` - only test
* `package` - only package

The rule of thumb is that `git clone && pnpm i && npm run build` should be almost the same as CI.

Reply:
Ok, that's how it works now, with the difference that:

- Both test and package tasks will run, beforehand, any dependencies they have. For example, the `package` task requires the projects to be compiled first

It shouldn't really affect performance much since the cache is quite aggressive.

Reply:
Sounds great!

Reply:
To be clear, build runs everything and **shouldn't** filter out hangar.

In CI, we can't use this command because we split up build, unit testing, and e2e testing, but the package.json command should still do everything

Reply:
I'm actually okay with filtering out hangar, it's just not what was previously requested

Reply:
Totally fine if we break this down in CI, but the leading principle is that "build" should mimick CI, and also include hangar.